<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Error</class>
 <widget class="QWidget" name="Error">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавть раздел</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/ico/pizza.ico</normaloff>images/ico/pizza.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fff;</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
background: #006EAC;
line-height: 21px;

text-align: center;
color: #FFFFFF;}

QPushButton:hover{
background-color:#005da7;
}
QPushButton:pressed{
background-color:#2D9CDB;
}</string>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameToProductEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #2D9CDB;</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
background: #006EAC;
line-height: 21px;

text-align: center;
color: #FFFFFF;}

QPushButton:hover{
background-color:#005da7;
}
QPushButton:pressed{
background-color:#2D9CDB;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameToLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
   </property>
   <property name="text">
    <string>Название товара</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
